src/migrations/index.js

```diff
@@ -0,0 +1,48 @@
+import database from '../databases';
+import log from '../log';
+import populateUploadedAt from './populateTokensUploadedAt';
+
+// eslint-disable-next-line no-process-env
+const APP_INSTANCE = process.env.NODE_APP_INSTANCE;
+
+function canNodeRunMigration() {
+  // Run when not in PM2 or on the first instance only
+  return !APP_INSTANCE || APP_INSTANCE === '0';
+}
+
+function startMigration(method) {
+  return new Promise(resolve => {
+    if (!canNodeRunMigration()) {
+      resolve();
+      return;
+    }
+    const reschedule = () => setTimeout(async () => {
```

Comment:
What's the reason for running this in a timeout? It seems a bit weird to block 2.5s on boot for that.